pkg/analyser/analysis.go

```diff
@@ -35,14 +35,15 @@ type Summary struct {
 }
 
 type Analysis struct {
-	unmanaged   []resource.Resource
-	managed     []resource.Resource
-	deleted     []resource.Resource
-	differences []Difference
-	summary     Summary
-	alerts      alerter.Alerts
-	Duration    time.Duration
-	Date        time.Time
+	unmanaged        []resource.Resource
+	managed          []resource.Resource
+	deleted          []resource.Resource
+	differences      []Difference
+	summary          Summary
+	alerts           alerter.Alerts
+	Duration         time.Duration
+	Date             time.Time
+	IgnoreRulesCount int
```

Comment:
I don't think this should be part of analysis struct

Reply:
Well the analysis object is what we use to transport every data about the scan, I'm afraid it's the best way to carry information along the execution

Reply:
check my latest commit, I put ignoreRulesCount value in a dedicated return value